### ISSUE TITLE
Fixtable fails to render with an error

### ISSUE THREAD

**Issue by NaeemTai** [MyPureCloud/fixtable-ember#109]
With the latests 3.2.0 version, fixtable sometimes fails to render with the following error:
```
ember.prod.js:16699 TypeError: sortedFilteredContent.map is not a function
    at Class.fixtableGrid$visibleContent (fixtable-grid.js:230)
    at ComputedPropertyPrototype.get (ember.prod.js:16086)
    at Object.get (ember.prod.js:20975)
    at RootPropertyReference.compute (ember.prod.js:14149)
    at RootPropertyReference.value (ember.prod.js:14024)
    at ArrayIterable.iterate (ember.prod.js:13788)
    at IterationArtifacts.isEmpty (ember.prod.js:41529)
    at IterablePresenceReference.value (ember.prod.js:45930)
    at ReferenceCache.initialize (ember.prod.js:42106)
    at ReferenceCache.peek (ember.prod.js:42080)
```

The previous version 3.1.0 was rendering all tables fine.

**Comment by smargovsky** [comment]
Hey @NaeemTai - is there anything common place about when you are seeing this?  Empty dataRows?  Filtering?  Sorting?  Or does it just seem random?

**Comment by danielsmeyerrr** [comment]
@smargovsky - It was happening every time you went to certain tabs in Supportability: Flows, and maybe (Queues and Audits, maybe?), for instance on the DCA 55bc org. It happened reliably, every time you went to that tab. We rolled back to fixtable-ember 3.1.0, so the issue is not appearing in DCA/TCA right now.

**Comment by NaeemTai** [comment]
@smargovsky , it seems random, but it was always happening with certain tables. The tables that are not rendering have filters and no sorting; there are other tables with the same traits that are rendering fine. So I don't see a trend between the tables that are not rendering that sets them apart from the ones that are rendering.

**Comment by michaelmcauley** [comment]
@NaeemTai, @danielsmeyerrr, @smargovsky:

I did some digging and discovered that on the pages where you had rendering issues, it was because the parent route was first setting the model (and thus Fixtable's `content` property) to an instance of `DS.Model` before the page controller kicks in and updates the model to an array of records. This wasn't causing a problem until changes in v3.2.0 explicitly assumed we were always dealing with an Array (or Array-like) object.

I've submitted a fix in pull request #110 and will update this issue once it's been approved & published. I've tested the Supportability app with these changes and it resolves your problem. Note, however, that Fixtable has been updated to output a warning when an unsupported type is supplied, so you'll be getting this warnings unless you update Supportability not to pass in junk while the page is loading 😜

**Comment by danielsmeyerrr** [comment]
Ok, good to know. Thanks for the quick turnaround on this @michaelmcauley !

**Comment by NaeemTai** [comment]
Thanks @michaelmcauley .

**Comment by michaelmcauley** [comment]
Fixed in v3.2.1
